### DISCUSSION TITLE
The filter masks in connection properties is poorly documented

### DISCUSSION THREAD

**Discussion by Austin1** [dbeaver/dbeaver#20573] [Q&A] [answered]
What do the filter wildcards really mean?

> NOTE: You can use masks with % and * to replace one or more symbols and _ to replace one symbol in the search combination.

From https://dbeaver.com/docs/wiki/Configure-Filters/

What does this really mean? Do % and * perform the exact same funciton? To they match on a single character or more than one character? Does the underscore just match symbols? How is a symbol defined? Is that any character is well and this means all three are just synonyms?

**Comment by daelynum** [comment, marked as answer]
Hello, @Austin1 

The `%` and `_` symbols are universal in SQL for multiple and single character substitution, respectively. However, the `*` is also used as a substitution character in some database systems and search systems. For instance, in Elasticsearch, `*` is used as a wildcard to match zero or more characters. We will update the [article](https://dbeaver.com/docs/wiki/Configure-Filters/) to make these distinctions clearer.
